src/bin/searchLogs.ts

```diff
@@ -1,14 +1,33 @@
+import { wordlists } from 'bip39'
 import nano from 'nano'
 
 import { config } from '../config'
+
+// const bip39b = require('bip39')
 const nanoDb = nano(config.couchDbFullpath)
 
-const searchTerms = process.argv.slice(2)
+// console.log(wordlists)
+// console.log(bip39b)
+// const searchTerms = process.argv.slice(2)
 
-if (searchTerms[0] == null) {
-  console.log('Missing search term')
-  process.exit(1)
-}
+// if (searchTerms[0] == null) {
+//   console.log('Missing search term')
+//   process.exit(1)
+// }
+
+// const checkme =
+//   'fault must outdoor spawn walk universe throw lonely embrace faculty bulb broken'
+
+// const w = checkme.split(' ')
+// let av = true
+// for (const word of w) {
+//   if (!wordlists.english.includes(word)) {
+//     av = false
+//     break
+//   }
+// }
+
+// console.log(`checkme = ${av.toString()}`)
```

Comment:
This commented out code is distracting and not necessary?

Reply:
done. this was a utility file for misc use but i cleaned this up anyway